okio/src/commonMain/kotlin/okio/internal/HashUtil.kt

```diff
@@ -36,6 +36,10 @@ internal infix fun UInt.leftRotate(bitCount: Int): UInt {
   return ((this shl bitCount) or (this shr (UInt.SIZE_BITS - bitCount))) and UInt.MAX_VALUE
 }
 
+internal inline infix fun Int.leftRotate(bitCount: Int): Int {
```

Comment:
Are we intending to move away from unsigned typed because they're experimental? 

We might be able to run some quick calculations to map our initial hash values and round constants from `UInt` to `Int` if that's the case.

Reply:
Yep.